### DISCUSSION TITLE
Official icons

### DISCUSSION THREAD

**Discussion by worksofliam** [codefori/vscode-ibmi#460] [Announcements]
Hey y'all. I am creating this locked thread to keep the latest logos here for future reference.

**Comment by worksofliam** [comment]
## Main icon

![Main Icon](https://user-images.githubusercontent.com/3708366/148324294-dc19c543-0ec1-4603-8d74-9cdfd5ee7f36.png)

**Comment by worksofliam** [comment]
## Unit Testing

![Unit Testing Icon (1)](https://user-images.githubusercontent.com/3708366/148324392-ac7621a8-b798-4d04-a256-ee76ba4e136d.png)

**Comment by worksofliam** [comment]
## Git icon

![Git Icon (1)](https://user-images.githubusercontent.com/3708366/148324420-48eddf67-2902-4ec4-9acf-0e84e790517f.png)

**Comment by worksofliam** [comment]
## Code Coverage

![Code Coverage Icon (1)](https://user-images.githubusercontent.com/3708366/148324441-94cf56f6-6dd0-4a4f-9405-650cbc8edeee.png)

**Comment by worksofliam** [comment]
## Db2 for i

![Db2 for i (1)](https://user-images.githubusercontent.com/3708366/172070449-dde4355c-4a89-480b-a07e-75ddb78e825f.png)

**Comment by worksofliam** [comment]
## Project Explorer

![Project Explorer](https://github.com/halcyon-tech/vscode-ibmi/assets/3708366/3f84a41c-de7d-4d89-ae55-20453ce19bbc)

**Comment by julesyan** [comment]
RPG
![rpg_final](https://github.com/user-attachments/assets/1dc897b2-4c1f-4f36-bcde-6b38b9be7cf9)

**Comment by julesyan** [comment]
CL
![cl_final](https://github.com/user-attachments/assets/9d77bdf7-f2e7-4d7e-bf3e-cbc04bf788ec)
